### PR TITLE
Ian all articles

### PR DESCRIPTION
**Pull request by LeeMellon**
Basic article display functions.

### CODE REVIEW

**Comment by LeeMellon** [comment]
Updated 7/17

File: src/app/login/login.component.ts [hunk 1]
```diff
@@ -1,33 +0,0 @@
-import { Component, OnInit } from '@angular/core';
```
Comment (Methodician) [inline src/app/login/login.component.ts]: Strange GitHub PR viewer is showing all the content of this file removed, and it won't let me edit it, saying I'm not on a branch. I can edit the other files using the online editor...

File: src/app/services/article.service.ts [hunk 2]
```diff
@@ -1,40 +1,81 @@
...
+      articleKeys.forEach(key => {
+          this.getArticleById(key.key).then(article => {
+          articlesList.push(article);
+          this.bookmarkedArticles$.next(articlesList);
+        });
+      });
+  });
+  }
+
+  getArticleUpdateTime(articleId) {
```
Comment (Methodician) [inline src/app/services/article.service.ts]: Is this actually going to the DB just to convert the date when getting an article?

Comment (Methodician) [inline src/app/services/article.service.ts]: Is this actually going to the DB just to convert the date when getting an article?

Comment (Methodician) [inline src/app/services/article.service.ts]: Is this actually going to the DB just to convert the date when getting an article?

**Pull request closed without merging**